content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,170 @@
+
+If a rule does not contain a scope name, it is evaluated for all values of that scope name.
+For example, if a rule does not contain the `repository` scope, it is evaluated for all repositories. If the rule scope is
+set to `always evaluate`, the rule is evaluated on all repositories and branches.
+
+{{< img src="ci/rule_scope_always_evaluate.png" alt="Rule scope for rules always evaluated" style="width:90%;">}}
+
+### Custom scope
+
+In addition to branch and repository, you can define custom scope names to further filter rules that are evaluated for a specific CI pipeline.
+
+Custom scopes can be added when creating a rule in the following way:
+1. Click on **+ Add Filter** and select **Custom Scope**
+2. Define the scope name, for example, `team`.
+3. Define the scope included or excluded values. This follows the same logic ([AM] ADD LINK TO SECTION ABOVE) defined for branch and repository.
+
+Differently from "branch" and "repository" scopes, custom scopes need to be provided in the `datadog-ci gate evaluate` command using the **--scope** option.
+For example, you can create a rule that is evaluated for the repository `example-repository` but only when the team is `backend`:
+
+{{< img src="ci/rule_scope_example_repository_team_backend.png" alt="Rule scope for example-repository and team backend" style="width:90%;">}}
+
+The rule is evaluated when this command is invoked in a CI pipeline of the `example-repository` repository:
+- `datadog-ci gate evaluate --scope team:backend`
+
+The rule is not evaluated when the following commands are invoked instead:
+- `datadog-ci gate evaluate`, as it does not define any team.
+- `datadog-ci gate evaluate --scope team:api --scope team:frontend`, as it defines teams different from "backend".
+
+## Edit a rule
+
+You can edit a Quality Gate rule by clicking on the edit icon in the [rules page][2]:
+
+{{< img src="ci/edit_quality_gate_rule.png" alt="Edit a Quality Gates rule" style="width:90%;">}}
+
+## Delete a rule
+
+You can delete a Quality Gate rule by clicking on the deletion icon in the [rules page][2]:
+
+{{< img src="ci/delete_quality_gate_rule.png" alt="Delete a Quality Gates rule" style="width:90%;">}}
+
+## Permissions
+
+The `quality_gate_rules_write` [permission][1] is required to create and edit Quality Gate rules.
+The `quality_gate_rules_read` [permission][1] is required to view Quality Gate rules.
+
+## Traceability
+
+You can view information about who created, modified, and deleted Quality Gates rules using Datadog's [Audit Trail][3] product.
+
+## Limitations during beta
+
+There are limitations for the beta version of the Quality Gate product:
+1. Currently only two rule types can be selected when creating rules: static analysis and tests.
+2. There are some rare cases in which the Quality Gates result are not correct, specifically in case of internal Datadog issues.
+
```

Comment:
We may not need to call out limitations in the beta (re: internal issues).

```suggestion
```

Reply:
Done!